site/src/components/RoleSelect/RoleSelect.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { screen } from "@testing-library/react"
+import {
+  assignableRole,
+  MockAuditorRole,
+  MockMemberRole,
+  MockOwnerRole,
+  MockTemplateAdminRole,
+  MockUserAdminRole,
+  render,
+} from "testHelpers/renderHelpers"
+import { RoleSelect } from "./RoleSelect"
+
+describe("UserRoleSelect", () => {
+  it("renders content", async () => {
+    // When
+    render(
+      <RoleSelect
+        roles={[
+          assignableRole(MockOwnerRole, false),
+          assignableRole(MockTemplateAdminRole, false),
+          assignableRole(MockAuditorRole, true),
+          assignableRole(MockUserAdminRole, true),
+        ]}
+        selectedRoles={[MockUserAdminRole, MockTemplateAdminRole, MockMemberRole]}
+        loading={false}
+        onChange={jest.fn()}
+        open={true}
```

Comment:
```suggestion
        open
```

hopefully that fixes lint for ya